.github/ISSUE_TEMPLATE/documentation_issue.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Documentation Issue
+about: Create a report to help us improve the documents
+labels: documentation
+---
+
+**Document URL.**
+What is the public documentation URL?
+
+**Does the document belong to this repo?**
+If no, please file the issue against the document repo (e.g. https://github.com/MicrosoftDocs/azure-docs/).
```

Comment:
Azure recommends opening issue by using the feedback button at the end of docs page, not directly through github. So lets add link to docs page here. and ask customers to open doc feedback there.
https://docs.microsoft.com/en-us/azure/azure-monitor/app/app-insights-overview